Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -183,6 +183,16 @@
 #import <pal/spi/cocoa/AVKitSPI.h>
 #endif
 
+#if USE(UIKIT_CONTEXT_MENU_INTERACTION)
+#import <UIKit/UITextInteractionAssistant_Private.h>
+#endif
+
+#if USE(UICONTEXTMENU)
+@interface WKContentView (ContextMenuAdditions)
+@property (nonatomic, readonly) UIContextMenuInteraction *contextMenuInteraction;
+@end
+#endif
```

Comment:
This is still being imported on non-internal builds.

Instead, it should go into the UIKitSPI.h inside WebKit when USE(APPLE_INTERNAL_SDK) is set, and fall back to forward declarations for non-internal builds.

Reply:
Looks like that header is already imported in UIKitSPI.h, so I can remove it from here